.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -95,7 +95,7 @@ body:
     attributes:
       label: Ghostty Configuration
       description: |
-        Please paste the output of `ghostty +show-config` here.
+        Please provide the minimum configuration needed to reproduce this issue.  If you cannot determine the minimum configuration needed to reproduce the issue, paste the output of `ghostty +show-config` here.
```

I'm not sure if it's useful to repeat the exact same noun phrase in the very next sentence, since it's not particularly easy to misinterpret it in this case.
```suggestion
        Please provide the minimum configuration needed to reproduce this issue.  If you cannot determine this, paste the output of `ghostty +show-config` here.
```